### DISCUSSION TITLE
It's possible to discuss epics/stories/issues within GitHub (linked directly)

### DISCUSSION THREAD

**Discussion by waltage** [gcivil-nyu-org/team-2-inperson#36] [General]
We're going to need to get used to having conversation threads on the issues themselves (and how the tagging interface works for code reviews)

Here's an example of an issue tag:
 (#5 ) -- just hash tag the issue number or click the "comment box with arrow" in the text box.

Here's an example of people tags:
@simaoalicechen @adelaidemz @yalexublue @Pooja-Codes @jainsvaibhav

**Comment by waltage** [comment]
When we have discussions like this, off to the right we can actually promote them into full issues; The discussions themselves can be tagged with corresponding issue labels; basically there are a lot of features available that keep everything integrated with where our artifacts will end up.

**Reply by simaoalicechen** [reply]
But first, answer the open issues I mentioned you on the SLACK.
